Assets/UI/WorldView/citybannermanager_CQUI_expansions.lua

```diff
@@ -0,0 +1,183 @@
+-- ===========================================================================
+-- CityBannerManager for Expansion 1 and Expansion 2
+-- ===========================================================================
+-- Functions and objects common to basegame and expansions
+include( "citybannermanager_CQUI.lua");
+
+-- ===========================================================================
+-- Cached Base Functions (Expansions only)
+-- ===========================================================================
+BASE_CQUI_CityBanner_Initialize = CityBanner.Initialize;
+BASE_CQUI_CityBanner_Uninitialize = CityBanner.Uninitialize;
+BASE_CQUI_CityBanner_UpdateInfo = CityBanner.UpdateInfo;
+BASE_CQUI_CityBanner_UpdatePopulation = CityBanner.UpdatePopulation;
+BASE_CQUI_CityBanner_UpdateRangeStrike = CityBanner.UpdateRangeStrike;
+BASE_CQUI_CityBanner_UpdateStats = CityBanner.UpdateStats;
```

Comment:
The unmodded game has CityBanner:Initialize and not CityBanner.Initialize, for the expansions. In Lua are they (: vs .) interchangeable?

Reply:
Okay figured out the difference, never mind.